dev/com.ibm.jbatch.container/bnd.bnd

```diff
@@ -115,12 +115,14 @@ instrument.classesExcludes: com/ibm/jbatch/container/internal/resources/*.class
 	com.ibm.websphere.appserver.spi.threading;version=latest,\
 	com.ibm.websphere.javaee.cdi.1.0;version=latest,\
 	com.ibm.websphere.appserver.api.kernel.service;version=latest,\
-	com.ibm.websphere.appserver.thirdparty.eclipselink;version=latest, \
+	com.ibm.websphere.appserver.thirdparty.eclipselink;version=latest,\
+	io.openliberty.persistence.3.1.thirdparty;version=latest,\
```

Comment:
don't know if you need io.openliberty.persistence.3.1.thirdparty in the buildpath now.